docs/plugins/pluggable_authentication.md

```diff
@@ -78,99 +78,17 @@ This will allow any username/password combination to successfully authenticate w
 
 With the permissive configuration working with irodsPamAuthCheck, the next step is to adjust your PAM configuration to your desired settings (LDAP, in this case). You will know that is correct when irodsPamAuthCheck behaves as you would expect when using LDAP username/passwords. iRODS uses irodsPamAuthCheck directly, so if it is working on the command line, it should work when run by iRODS.
 
-### Server SSL Setup
+### Setting up SSL/TLS
 
 Since PAM requires the user's password in plaintext, iRODS relies on SSL encryption to protect these credentials.  PAM authentication makes use of SSL regardless of the iRODS Zone SSL configuration (meaning even if iRODS explicitly does *not* encrypt data traffic, PAM will use SSL during authentication).
 
-In order to use the iRODS PAM support, you also need to have SSL working between the iRODS client and server. The SSL communication between client and iRODS server needs some basic setup in order to function properly. Much of the setup concerns getting a proper X.509 certificate setup on the server side, and setting up the trust for the server certificate on the client side. You can use either a self-signed certificate (best for testing) or a certificate from a trusted CA.
+In order to use the iRODS PAM support, you also need to have SSL working between the iRODS client and server.
 
-Here are the basic steps to configure the server:
+See [SSL/TLS Documentation](../../system_overview/ssl_and_tls) for instructions to set up SSL/TLS communications between iRODS clients and servers.
 
-#### Generate a new RSA key
+After setting up SSL on the server side, test SSL by using the PAM authentication (which requires an SSL connection) and running ``iinit`` with the log level set to LOG_NOTICE.
```

Comment:
`imiscsvrinfo` is probably a better fit for confirming TLS communication.

Reply:
Maybe we should just remove this entire debugging section... Using `IRODS_LOG_LEVEL` environment variable surely isn't going to be supported for much longer as it likely employs `rodsLog`...

Reply:
The log level stuff should be removed, but a mention of `imiscsvrinfo` seems like a good idea since they can just run it without authenticating and confirm that TLS is enabled.

That highlights our efforts around building better tools for users.

Reply:
Maybe we can add a "Verification and Debugging" section to the SSL/TLS page which highlights that?

Reply:
Sure, that sounds good to me.

Reply:
Added new section. Can continue discussion in a new convo "over there" if needed.